mne_g3d/viz.py

```diff
@@ -0,0 +1,127 @@
+import logging
+
+import ipyvolume as ipv
+from ipyvolume.pylab import style
+import nibabel
+import numpy as np
+
+def read_brain_mesh(surface_path: str):
+    """Function reads triangular format Freesurfer brain surface.
+
+    Parameters
+    ----------
+    surface_path: :obj: `str`
+        Path to the brain surface.
+
+    Returns
+    -------
+    :obj: `numpy.array`
+        Array of vertex (x, y, z) coordinates, of size number_of_vertices x 3.
+    :obj: `numpy.array`
```

Comment:
This should follow our standard docstring conventions (see how we do it in MNE). For now don't bother with `:obj:`, eventually this is something that should be handled by `numpydoc`

Reply:
I will check MNE docstyles, you follow Numpy doc styles, don't you? I don't quite understand what you have mentioned about `:obj:`, should I just skip this string?

Reply:
Yes do not add it. And for `Returns` we always give names even though they aren't required by the numpydoc standard